bot/exts/fun/uwu.py

```diff
@@ -178,19 +181,29 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
         await clean_content(fix_channel_mentions=True).convert(ctx, text)
 
         fun_cog: t.Optional[Fun] = ctx.bot.get_cog("Fun")
-        if fun_cog:
-            text, embed = await fun_cog._get_text_and_embed(ctx, text)
 
-            # Grabs the text from the embed for uwuification.
-            if embed is not None:
-                embed = fun_cog._convert_embed(self._uwuify, embed)
+        if fun_cog:
+            # Grabs the text from the embed for uwuification
+            if embeds:
+                embed = messages.convert_embed(self._uwuify, embeds[0])
+            else:
+                # Parse potential message links in text
+                text, embed = await messages.get_text_and_embed(ctx, text)
+
+                # If an embed is found, grab and uwuify its text
+                if embed:
+                    embed = messages.convert_embed(self._uwuify, embed)
```

Comment:
```suggestion
        # Grabs the text from the embed for uwuification
        if embeds:
            embed = messages.convert_embed(self._uwuify, embeds[0])
        else:
            # Parse potential message links in text
            text, embed = await messages.get_text_and_embed(ctx, text)

            # If an embed is found, grab and uwuify its text
            if embed:
                embed = messages.convert_embed(self._uwuify, embed)
```
We no longer need `fun_cog` and related checks since the functions were moved to `messages.py`